numpy/linalg/_linalg.pyi

```diff
@@ -373,74 +373,74 @@ def lstsq(a: _ArrayLikeComplex_co, b: _ArrayLikeComplex_co, rcond: float | None
 @overload
 def norm(
     x: ArrayLike,
-    ord: float | L["fro", "nuc"] | None = ...,
-    axis: None = ...,
-    keepdims: bool = ...,
+    ord: float | L["fro", "nuc"] | None = None,
+    axis: None = None,
+    keepdims: bool = False,
 ) -> floating: ...
 @overload
 def norm(
     x: ArrayLike,
-    ord: float | L["fro", "nuc"] | None = ...,
-    axis: SupportsInt | SupportsIndex | tuple[int, ...] = ...,
-    keepdims: bool = ...,
+    ord: float | L["fro", "nuc"] | None = None,
+    axis: SupportsInt | SupportsIndex | tuple[int, ...] = None,
+    keepdims: bool = False,
 ) -> Any: ...
 
 @overload
 def matrix_norm(
     x: ArrayLike,
     /,
     *,
-    ord: float | L["fro", "nuc"] | None = ...,
-    keepdims: bool = ...,
+    ord: float | L["fro", "nuc"] | None = "fro",
+    keepdims: bool = False,
 ) -> floating: ...
 @overload
 def matrix_norm(
     x: ArrayLike,
     /,
     *,
-    ord: float | L["fro", "nuc"] | None = ...,
-    keepdims: bool = ...,
+    ord: float | L["fro", "nuc"] | None = "fro",
+    keepdims: bool = False,
 ) -> Any: ...
 
 @overload
 def vector_norm(
     x: ArrayLike,
     /,
     *,
-    axis: None = ...,
-    ord: float | None = ...,
-    keepdims: bool = ...,
+    axis: None = None,
+    ord: float | None = 2,
+    keepdims: bool = False,
 ) -> floating: ...
 @overload
 def vector_norm(
     x: ArrayLike,
     /,
     *,
-    axis: SupportsInt | SupportsIndex | tuple[int, ...] = ...,
-    ord: float | None = ...,
-    keepdims: bool = ...,
+    axis: SupportsInt | SupportsIndex | tuple[int, ...] = None,
```

Comment:
```suggestion
    axis: SupportsInt | SupportsIndex | tuple[int, ...],
```

Reply:
thanks for flagging this, but isn't `None` the default? better to add `| None` as an option then? 

https://github.com/numpy/numpy/blob/a80542541130a2dc47eab2a6972019453cc0c518/numpy/linalg/_linalg.py#L3514